src/components/settings-menu/index.tsx

```diff
@@ -47,13 +47,13 @@ export class SettingsMenu extends React.Component<Properties, State> {
   };
 
   renderSettingsHeader() {
-    const c = bem('header');
+    const cn = bemClassName('header');
```

Comment:
Since we're in the groove here the next step should probably be to move this to the top of the file so we're not redefining functions on every render. Also, the name doesn't match convention and would probably be pretty trivial to change?

Reply:
Yeah sounds good I'll revisit this one in a follow up